docs/eloquent-models/relationships.txt

```diff
@@ -0,0 +1,529 @@
+          space_explorer_ids: [ '65e1043b5265269a03078ad4', '65e1043b5265269a03078ad5' ]
+        },
+        {
+          _id: ObjectId('65e1043b5265269a03078ad2'),
+          name: 'Jupiter',
+          // ...
+          space_explorer_ids: [ '65e1043b5265269a03078ad3', '65e1043b5265269a03078ad5' ]
+        }
+      ]
+
+      // Documents in the "space_explorers" collection
+      [
+        {
+          _id: ObjectId('65e1043b5265269a03078ad3'),
+          name: 'Tanya Kirbuk',
+          // ...
+          planet_ids: [ '65e1043a5265269a03078ad0', '65e1043b5265269a03078ad2' ]
+        },
+        {
+          _id: ObjectId('65e1043b5265269a03078ad4'),
+          name: 'Mark Watney',
+          // ...
+          planet_ids: [ '65e1043a5265269a03078ad0', '65e1043a5265269a03078ad1' ]
+        },
+        {
+          _id: ObjectId('65e1043b5265269a03078ad5'),
+          name: 'Jean-Luc Picard',
+          // ...
+          planet_ids: [
+            '65e1043a5265269a03078ad0',
+            '65e1043a5265269a03078ad1',
+            '65e1043b5265269a03078ad2'
+          ]
+        }
+      ]
+
+The following sample code shows how to access the related models by using
+the dynamic properties as defined in the example classes.
+
+.. literalinclude:: /includes/eloquent-models/relationships/RelationshipController.php
+   :language: php
+   :dedent:
+   :start-after: begin many-to-many dynamic property example
+   :end-before: end many-to-many dynamic property example
+
+.. _laravel-embedded-document-pattern:
+
+Embedded Document Pattern
+-------------------------
+
+In MongoDB, the embedded document pattern adds the related model's data into
+the parent model instead of keeping foreign key references. Use this pattern
+to meet one or more of the following requirements:
+
+- Keeping associated data together in a single collection
+- Performing atomic updates on multiple fields of the document and the associated
+  data
+- Reducing the number of reads required to fetch the data
+
+In {+odm-short+}, you can define embedded documents by using one of the
+following dynamic property methods:
+
+- ``embedsOne()`` to embed a single document
+- ``embedsMany()`` to embed multiple documents
+
+.. note::
+
+   These methods return Eloquent collections, which differ from query builder
+   objects.
+
+To learn more about the MongoDB embedded document pattern, see the following
+MongoDB server tutorials:
+
+- :manual:`Model One-to-One Relationships with Embedded Documents </tutorial/model-embedded-one-to-one-relationships-between-documents/>`
+- :manual:`Model One-to-Many Relationships with Embedded Documents </tutorial/model-embedded-one-to-many-relationships-between-documents/>`
+
+The following section shows an example of how to use the embedded document
+pattern.
+
+Embedded Document Example
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example class shows how to define an ``embedsMany`` one to many
+relationship between a ``SpaceShip`` and ``Cargo`` model:
```

Comment:
S; related to earlier comment - decide on a convention to describe this relationship.
```suggestion
The following example class shows how to define an ``EmbedsMany`` one to many
relationship between a ``SpaceShip`` and ``Cargo`` model:
```
Or
```suggestion
The following example class shows how to use the ``embedsMany()`` method to define a one to many
relationship between a ``SpaceShip`` and ``Cargo`` model:
```

Reply:
Using the standardized version mentioned in [this comment)(https://github.com/mongodb/laravel-mongodb/pull/2747#discussion_r1516419165).